### PR TITLE
♻️  CI scripts refactoring

### PR DESCRIPTION
**Pull request by amortemousque**
## Motivation

During the new staging ci process development utils have been introduced. 
This PR updates all the scripts which can have a use for utils.

## Changes

- Update scripts

## Testing

Locally, Gitlab

---

I have gone over the [contributing](https://github.com/DataDog/browser-sdk/blob/main/CONTRIBUTING.md) documentation.

### CODE REVIEW

File: scripts/bs-wrapper.js [hunk 1]
```diff
@@ -52,13 +46,15 @@ function hasRunningBuild() {
...
 }
 
 function timeout(ms) {
   return new Promise((resolve) => setTimeout(resolve, ms))
 }
+
+main().catch((error) => {
+  printError('\nStacktrace:\n', error)
+  process.exit(1)
+})
```
Comment (bcaudan) [inline scripts/bs-wrapper.js:60]: We are doing that a lot, what about extracting it also?
maybe something like 
```ts
printFormattedError(error)
process.exit(1)
```
or
```ts
catch(logAndExit)
```

Comment (amortemousque) [reply scripts/bs-wrapper.js:60]: Yes, sure. I'll do the second proposition

File: scripts/replace-build-env.js [hunk 2]
```diff
@@ -9,21 +10,20 @@ const buildEnv = require('./build-env')
...
-  console.log(
+  printLog(
     'Changed files:',
     results.filter((entry) => entry.hasChanged).map((entry) => entry.file)
   )
   process.exit(0)
 } catch (error) {
-  console.error('Error occurred:', error)
+  printError('\nStacktrace:\n', error)
   process.exit(1)
```
Comment (bcaudan) [inline scripts/replace-build-env.js:28]:
```suggestion
  logAndExit(error)
```

**Review by bcaudan — APPROVED** [review]

File: scripts/replace-build-env.js (hunk 2 above)
Comment (amortemousque) [reply scripts/replace-build-env.js]: I've changed the design of `replace-build-env.js` to be like the other scripts

**Review by BenoitZugmeyer — APPROVED** [review]

**Pull request merged**